spec.html

```diff
@@ -35960,6 +35960,9 @@ <h1>
           <emu-note>
             <p>The listed properties form a superset of what <a href="https://unicode.org/reports/tr18/#RL1.2">UTS18 RL1.2</a> requires.</p>
           </emu-note>
+          <emu-note>
+            <p>The spellings of entries in these tables (including casing) were chosen to match the first occurrence of each property in the file <a href="https://unicode.org/Public/UCD/latest/ucd/PropertyAliases.txt"><code>PropertyAliases.txt</code></a> in the Unicode Character Database at the time each entry was added to this specification. However, because the precise spellings in those files are not guaranteed to be stable, implementations are required to follow this table rather than those files.</p>
```

Comment:
Wait, aren't spellings guaranteed to be stable now?

Reply:
> Wait, aren't spellings guaranteed to be stable now?

Yes --> https://github.com/tc39/ecma262/pull/2649#issuecomment-1079239428

Reply:
Oh, I guess we should just delete this note?

Reply:
Or modify per https://github.com/tc39/ecma262/pull/2649#discussion_r799844489. Either is fine by me.

Reply:
How about this? https://github.com/tc39/ecma262/pull/2649/commits/e7496c38c3742a5ae4c40784593755a9b4a91925

Reply:
Works for me.